rmw_opendds_cpp/include/rmw_opendds_cpp/opendds_static_client_info.hpp

```diff
@@ -30,4 +30,4 @@ struct OpenDDSStaticClientInfo
 };
 }  // extern "C"
 
-#endif  // RMW_OPENDDS_CPP__CONNEXT_STATIC_CLIENT_INFO_HPP_
+#endif  // RMW_OPENDDS_CPP__OPENDDS_STATIC_CLIENT_INFO_HPP_
```

Comment:
(This is a comment for lines 27-28. I don't know how to add a comment to those lines directly. Feel free to move it.) Should we use *_var types for DataReader and ReadCondition?

Reply:
The proper implementation of client is not in the scope of this PR. See the PR comment 2 at the top.